internal/internal_worker_deployment_client.go

```diff
@@ -29,6 +30,15 @@ func safeAsTime(timestamp *timestamppb.Timestamp) time.Time {
 	}
 }
 
+// safeAsDuration ensures that a nil proto duration makes `AsDuration()` return 0.
+func safeAsDuration(duration *durationpb.Duration) time.Duration {
```

Comment:
Why was this added? `.AsDuration()` should already be safe for nil

Reply:
> Why was this added? `.AsDuration()` should already be safe for nil

thanks for review, I will check this soon

Reply:
> Why was this added? `.AsDuration()` should already be safe for nil

`.AsDuration` is a func on `*durationpb.Duration`, and will panic if call it from nil varible

![image](https://github.com/user-attachments/assets/454ccd50-3406-41b8-8523-828464ac0c5e)

![image](https://github.com/user-attachments/assets/ffaa6b94-e09d-4edf-81ca-0c11609248f4)

Reply:
>and will panic if call it from nil varible,

That is not true

```
	var d *durationpb.Duration
	fmt.Println(d.AsDuration())
```

prints 0s

> so it is like the .AsTime

That is also not true